pandas/tests/window/moments/test_moments_ewm.py

```diff
@@ -274,15 +274,15 @@ def test_ew_min_periods(self, min_periods, name):
         assert result2.dtype == np.float_
 
 
-class TestEwmMomentsConsistency(ConsistencyBase):
+class TestEwmMomentsConsistency(Base):
     def setup_method(self, method):
         self._create_data()
 
     def test_ewmcov_pairwise(self):
-        self._check_pairwise_moment("ewm", "cov", span=10, min_periods=5)
+        check_pairwise_moment(self.frame, "ewm", "cov", span=10, min_periods=5)
```

Comment:
Could we turn these two test into 1 using `pytest.mark.parameterize(func, ['cov', 'corr'])`? Think there's similar potential below

Reply:
thanks, good catch! 
parametrized! also for that two tests in `rolling`